--- setup.py	2015-06-18 21:49:31.000000000 -0700
+++ setup.py.fixed	2015-06-18 21:51:43.000000000 -0700
@@ -236,6 +236,7 @@
         unix = "DARWIN"
 
         self.extra_compile_args.append("-fvisibility-inlines-hidden")
+        self.extra_compile_args.append("-Wno-error=unused-command-line-argument-hard-error-in-future")
         self.extra_compile_args.append("-DCASE_INSENSITIVE")
 
         if unameOut[2][0] == "8":
